src/lib/features/project/project-service.ts

```diff
@@ -218,6 +222,12 @@ export default class ProjectService {
         return projects;
     }
 
+    async addOwnersToProjects(
+        projects: IProjectWithCount[],
+    ): Promise<IProjectWithCount[]> {
+        return this.projectOwnersReadModel.addOwners(projects);
+    }
```

Comment:
Why are we adding this to the service? I thought the controller was gonna call the read model directly? Maybe I've misunderstood? 🤔

Reply:
I don't think we have this pattern anywhere. Controllers use stores always with the help of services

Reply:
The playground controller (`src/lib/features/playground/playground.ts`) calls the view model directly, so that looks like the pattern to me? Is it different in enterprise or something?